### PR TITLE
[MOSIP-28460] added resident and 1.2.0.1 versions

### PR DESCRIPTION
**Pull request by ckm007**

### CODE REVIEW

File: postgres-upgrade/Dockerfile [hunk 1]
```diff
@@ -15,6 +15,7 @@ COPY ./repos/partner-management-services/db_upgrade_scripts .
 COPY ./repos/admin-services/db_upgrade_scripts .
 COPY ./repos/keymanager/db_upgrade_scripts .
 COPY ./repos/audit-manager/db_upgrade_scripts .
+COPY ./repos/resident-services/db_upgrade_scripts .
```
Comment (vishwa-vyom) [inline postgres-upgrade/Dockerfile:18]: hotlist is missing ?

File: postgres-upgrade/upgrade.csv [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+mosip_master,1.2.0.1-B1,1.2.0.1,upgrade
 mosip_keymgr,1.1.5.5,1.2.0.1-B1,upgrade
+mosip_keymgr,1.2.0.1-B1,1.2.0.1-B2,upgrade
+mosip_keymgr,1.2.0.1-B2,1.2.0.1,upgrade
 mosip_audit,1.1.5.5,1.2.0.1-B1,upgrade
+mosip_audit,1.2.0.1-B1,1.2.0.1,upgrade
+mosip_resident,1.1.5.5,1.2.0.1-B1,upgrade
+mosip_resident,1.2.0.1-B1,1.2.0.1-B2,upgrade
+mosip_residet,1.2.0.1-B2,1.2.0.1,upgrade
+mosip_residet,1.2.0.1-B2,1.2.0.1,upgrade
```
Comment (vishwa-vyom) [inline postgres-upgrade/upgrade.csv:26]: spell check

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: postgres-upgrade/Dockerfile (hunk 1 above)
Comment (ckm007) [reply postgres-upgrade/Dockerfile:18]: hotlist is part of ./repos/admin-services/db_upgrade_scripts . 
so it is not added.

File: postgres-upgrade/upgrade.csv (hunk 2 above)
Comment (ckm007) [reply postgres-upgrade/upgrade.csv]: updated.
also added few more missing db upgrades

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**